topics/persistence.md

```diff
@@ -168,6 +176,8 @@ cases if you want, but the default configuration is to continue regardless
 the fact the last command in the file is not well-formed, in order to guarantee
 availability after a restart.
 
+Since Redis 7.0.0, only the last AOF file (BASE or INCR) can be truncated. Otherwise redis will exit even if aof-load-truncated is enabled.
```

Comment:
i don't think we need to mention it.. logically the tail of the AOF is just the last file.

Reply:
OK